#!/usr/bin/env python

PACKAGE="baxter_test"

from dynamic_reconfigure.parameter_generator_catkin import *
import math

gen=ParameterGenerator()

gen.add("angle_rotation", double_t, 0, "Angle rotation on Z", 0,-math.pi,math.pi)
gen.add("marker_x",double_t,0,"Displacement on X axis",0,-1.0,1.0)
gen.add("marker_y",double_t,0,"Displacement on Y axis",0,-1.0,1.0)
gen.add("target_x",double_t,0,"Target displacement on X axis",0,-0.1,0.1)
gen.add("target_y",double_t,0,"Target displacement on Y axis",0,-0.1,0.1)

exit(gen.generate(PACKAGE, "baxter_test", "rotation"))